docs/source/architecture/upgradeable_proxy_contracts.rst

```diff
@@ -1,28 +1,35 @@
-# Nucypher's Approaches to Upgradeable Contracts
+Nucypher's Approaches to Upgradeable Contracts
+==============================================
 
 Smart contracts in Ethereum are immutable...
 Even if a contract can be deleted, it still exists in the blockchain after `selfdestruct`, and only the storage is cleared.
 In order to fix bugs and provide upgrade logic it is possible to change the contract (address) and save the original contract's storage values.
 
 
-## Approach A
+Approach A
+----------
 
 One simple way to achieve this is to create a new contract, copy the original storage values to the new contract, then self-destruct (mark as deleted) the old contract.
 When this happens, the client changes the address used for a requested contract.
 
-*Note: (There will be two deployed versions of the contract during storage migration)*
+.. note::
 
+  There will be two deployed versions of the contract during storage migration)*
```

Comment:
```suggestion
  There will be two deployed versions of the contract during storage migration)
```

Reply:
> What is your opinion on moving the `NU` image to the sidebar like in this example?
> 
> ![image](https://user-images.githubusercontent.com/2382226/51880383-ec81b480-232b-11e9-8984-64907427fa38.png)

Sidebar logo is was cleaner. I'll include this in the next docs PR

Reply:
Looks better indeed